.eslintrc.json

```diff
@@ -0,0 +1,117 @@
+{
+  "parser": "babel-eslint",
+  "extends": [
+    "standard",
+    "standard-react"
+  ],
+  "env": {
+    "browser": true,
+    "jest": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module"
+  },
+  "rules": {
+    "jest/no-disabled-tests": "error",
+    "jest/no-focused-tests": "error",
+    "jest/no-identical-title": "error",
+    "jest/prefer-to-have-length": "error",
+    "jest/prefer-to-be-null": "error",
+    "jest/valid-expect": "error",
+    "jest/valid-describe": "error",
+    "jest/no-test-prefixes": "error",
+    "jest/consistent-test-it": "error",
+    "react/jsx-uses-react": "error",
+    "react/jsx-uses-vars": "error",
+    "react/jsx-indent": "off",
+    "react/jsx-indent-props": "off",
+    "react/prop-types": "error",
+    "react/jsx-tag-spacing": "error",
+    "react/no-unused-prop-types": "error",
+    "react/no-access-state-in-setstate": "error",
+    "react/prefer-es6-class": [
+      "error",
+      "always"
+    ],
+    "react/no-deprecated": "error",
+    "react/no-did-mount-set-state": "error",
+    "react/no-did-update-set-state": "error",
+    "react/no-direct-mutation-state": "error",
+    "react/no-typos": "error",
+    "sort-imports-es6-autofix/sort-imports-es6": [
+      "off",
+      {
+        "ignoreCase": false,
+        "ignoreMemberSort": false,
+        "memberSyntaxSortOrder": [
+          "none",
+          "all",
+          "single",
+          "multiple"
+        ]
+      }
+    ],
+    "arrow-parens": [
+      "error",
+      "always"
```

Comment:
I'd actually prefer "as-needed" here. Too many parentheses decrease readability I think.

Reply:
I'm not sure if its more readable for beginners. Imo Having something like `A => B => {}` is at first a wtf. Maybe this helps.

Reply:
But with parentheses this would be `(A) => (B) => {}` which is not better, is it ? 🤔

Reply:
I have to agree with @Schedulaar, if you have only one argument it's imo more readable without parentheses